curvefs/src/client/rpcclient/metacache.h

```diff
@@ -208,6 +200,7 @@ class MetaCache {
     RWLock rwlock4Partitions_;
     PartitionInfoList partitionInfos_;
     RWLock rwlock4copysetInfoMap_;
+    // TODO(Xinlong-Chen): delete appliedindex from CopysetInfo
```

Comment:
When will this problem be fixed?

Reply:
`CopysetInfo` is defined in `metacache_struct.h` in curvebs. If I change this struct, it will cause some ci problems, so I will fix it when prs merge. It's not difficult to do it.

https://github.com/opencurve/curve/blob/2fa67c8961fbb48eac5254c1ce75ce9200bcf989/curvefs/src/client/rpcclient/metacache.h#L38